Src/Plugins/Security/Security.csproj

```diff
@@ -31,6 +31,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="aliyun-net-sdk-core" Version="1.5.10" />
```

Comment:
>aliyun-net-sdk-core [](start = 31, length = 19)

why do we need a package and the dll itself?

Reply:
Neither the nuget package nor the dll have all of the classes necessary to make the code compile.  I also tried adding other Nuget packages and they were missing some of the classes used.  I don't know what to tell you.

---
In reply to: [591818226](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591818226) [](ancestors = 591818226)